tests/cupy_tests/creation_tests/test_from_data.py

```diff
@@ -286,7 +286,7 @@ def test_array_copy_list_of_cupy_with_dtype_char(self, xp, dtype1, dtype2,
     @testing.numpy_cupy_array_equal()
     def test_array_no_copy(self, xp, dtype, order):
         a = testing.shaped_arange((2, 3, 4), xp, dtype)
-        b = xp.array(a, copy=False, order=order)
+        b = xp.array(a, copy=None, order=order)
```

Comment:
These tests assume that `b` is not copied from `a`. Could you please revert these changes?

Reply:
This makes tests stricter, but OK, reverted.